draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -194,6 +194,12 @@ of their proposals for publication in the RFC series to allow others
 to concretely understand and investigate the wealth of proposals in
 this space.
 
+This document is meant to reduce the barriers to entry for new congestion
+control work. As such, proponents should not interpret these criteria as a
+checklist of requirements before approaching the IETF. Instead, proponents
+are encouraged to have thought about most of these issues beforehand, and have
+the willingness to do the work implied below.
+
```

Comment:
The work below, seems to me to suggest the next section, which isn't quite what I had hoped. Perhaps we need a different word?